apps/system-e2e/src/tests/islandis/admin-portal/smoke/access-control.spec.ts

```diff
@@ -12,8 +12,10 @@ test.describe('Admin portal access control', () => {
     contextGranter = await session({
       browser,
       storageState: 'service-portal-faereyjar.json',
+      delegation: '65° ARTIC ehf.',
```

Comment:
Note that this will break on `staging`, since the name is different. If you omit this parameter, we choose the first delegation available (which incidentally is always the `65...` delegation)